v1.17-5.0/Dockerfile

```diff
@@ -0,0 +1,82 @@
+FROM ruby:3.2.6-alpine3.20 as base
+
+ARG BUILD_DEPS=" \
+      make gcc g++ libc-dev \
+      wget bzip2 zlib-dev git linux-headers \
+      automake autoconf libtool build-base \
+      ruby-dev libc6-compat geoip-dev \
+      snappy-dev gnupg bash openssl-dev \
+      "
+
+RUN addgroup -S -g 101 fluent && adduser -S -G fluent -u 100 fluent \
+ # for log storage (maybe shared with host)
+ && mkdir -p /fluentd/log \
+ # configuration/plugins path (default: copied from .)
+ && mkdir -p /fluentd/etc /fluentd/plugins \
+ && chown -R fluent /fluentd && chgrp -R fluent /fluentd \
+ && mkdir -p /buffers && chown -R fluent /buffers \
+ && apk add --no-cache \
+        ca-certificates \
+        ruby ruby-irb ruby-etc \
+        tini libmaxminddb geoip \
+        snappy \
+ && apk add --no-cache $BUILD_DEPS \
+ && apk add --no-cache "libssl3>=3.1.4-r1" \
+ && apk add --no-cache "libcrypto3>=3.1.4-r1" \
```

Comment:
I think we can get rid of these now as well.
```sh
/ # apk search libcrypto3
libcrypto3-3.3.2-r1
```

```sh
/ # apk search libssl3
libssl3-3.3.2-r1
nss-3.105-r0
```

Reply:
thx, updated